### PR TITLE
ADOP-1130 Check Your Answers- Part-3

### PR DESCRIPTION
**Pull request by mohitvijay45**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ADOP-1130

### Change description ###
LA Portal - Review your answers functionality - Part 3 - Backend Validation (same as CUI)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/adoption-cos-api/values.preview.template.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
   environment:
-    CASE_DATA_STORE_BASEURL: http://ccd-data-store-api-adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal
-    CASE_DOCUMENT_AM_BASEURL: http://ccd-case-document-am-api-adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal
+    CASE_DATA_STORE_BASEURL: https://ccd-data-store-api-adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal
```
Comment (adusumillipraveen) [inline charts/adoption-cos-api/values.preview.template.yaml:8]: You could use svc URLs here like every product chart deployment does.

```suggestion
    CASE_DATA_STORE_BASEURL: http://adoption-cos-api-pr-${CHANGE_ID}-ccd-data-store-api
```

File: charts/adoption-cos-api/values.preview.template.yaml [hunk 2]
```diff
@@ -5,8 +5,8 @@ java:
...
+    CASE_DOCUMENT_AM_BASEURL: https://ccd-case-document-am-api-adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal
```
Comment (adusumillipraveen) [inline charts/adoption-cos-api/values.preview.template.yaml:9]:
```suggestion
    CASE_DOCUMENT_AM_BASEURL: http://adoption-cos-api-pr-${CHANGE_ID}-ccd-case-document-am-api
```

File: charts/adoption-cos-api/values.preview.template.yaml [hunk 3]
```diff
@@ -53,12 +53,12 @@ ccd:
   global:
     idamApiUrl: https://idam-api.aat.platform.hmcts.net
     idamWebUrl: https://idam-web-public.aat.platform.hmcts.net
-    dmStoreUrl: http://dm-store-aat.service.core-compute-aat.internal
+    dmStoreUrl: https://dm-store-aat.service.core-compute-aat.internal
```
Comment (adusumillipraveen) [inline charts/adoption-cos-api/values.preview.template.yaml:56]:
```suggestion
    dmStoreUrl: http://dm-store-aat.service.core-compute-aat.internal
```

File: charts/adoption-cos-api/values.preview.template.yaml [hunk 4]
```diff
@@ -53,12 +53,12 @@ ccd:
...
     ccdAdminWebIngress: admin-web-${SERVICE_FQDN}
     ccdApiGatewayIngress: gateway-${SERVICE_FQDN}
     postgresUsername: hmcts
     postgresPassword: hmcts
-    s2sUrl: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+    s2sUrl: https://rpe-service-auth-provider-aat.service.core-compute-aat.internal
```
Comment (adusumillipraveen) [inline charts/adoption-cos-api/values.preview.template.yaml:61]:
```suggestion
    s2sUrl: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
```

File: charts/adoption-cos-api/values.preview.template.yaml [hunk 5]
```diff
@@ -205,7 +205,7 @@ ccd-case-document-am-api:
     environment:
       CASE_DOCUMENT_S2S_AUTHORISED_SERVICES: ccd_case_document_am_api,ccd_gw,xui_webapp,ccd_data,bulk_scan_processor,dg_docassembly_api,bulk_scan_orchestrator,adoption_cos_api,adoption_web
       DM_STORE_BASE_URL: http://dm-store-aat.service.core-compute-aat.internal
-      CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api-adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal
+      CCD_DATA_STORE_API_BASE_URL: https://ccd-data-store-api-adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal
       IDAM_USER_URL: https://idam-api.aat.platform.hmcts.net
-      IDAM_S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+      IDAM_S2S_URL: https://rpe-service-auth-provider-aat.service.core-compute-aat.internal
```
Comment (hannah38) [inline charts/adoption-cos-api/values.preview.template.yaml:210]: Hey, another aat one that needs to stay http
```suggestion
      IDAM_S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
```

**Pull request closed without merging**